octoprint_discordremote/command.py

```diff
@@ -25,6 +28,9 @@ def __init__(self, plugin):
         self.command_dict['/help'] = {'cmd': self.help, 'description': "Print this help."}
         self.command_dict['/pause'] = {'cmd': self.pause, 'description': "Pause current print."}
         self.command_dict['/resume'] = {'cmd': self.resume, 'description': "Resume current print."}
+        self.command_dict['/timelapse'] = {'cmd': self.timelapse, 'description': "List all timelapses and download links."}
+        self.command_dict['/getfile'] = {'cmd': self.get_file, 'params': "{location} {filename}",
```

Comment:
@OctopusProteins And now that I think about it, we could merge /files and /getfile fairly easily as well, and make them a bit more consistent. I am happy to merge without this change though, and I can do it?

Reply:
Sure, go ahead ;)